lib/src/components/NxThreatCounter/NxThreatCounter.scss

```diff
@@ -54,16 +54,20 @@
 }
 
 .nx-threat-counter-container {
-  display: inline-grid;
+  // This is the calculated width (rounded up) of the
+  // longest threat counter, "unspecified".
+  $min-width: 154px;
+
+  display: grid;
   gap: var(--nx-spacing-4x);
-  grid-template-columns: repeat(5, 1fr);
+  grid-template-columns: repeat(auto-fill, minmax($min-width, 1fr));
   margin: var(--nx-spacing-4x) 0 var(--nx-spacing-6x) 0;
 }
 
 .nx-threat-counter-container--column {
-  grid-template-columns: auto;
+  grid-template-columns: min-content;
 }
 
 .nx-threat-counter-container--grid {
-  grid-template-columns: auto auto;
+  grid-template-columns: min-content min-content;
```

Comment:
I don't think it was previously believed that the grid mode was buggy.  However, looking closer at it now, I see that it is in that the two columns are supposed to be the same width but they aren't.  But the change you have here does not fix or change that.